docs/pages/web/textarea.js

```diff
@@ -635,6 +638,36 @@ function Example(props) {
     </Box>
   );
 }
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="With rows"
+          description={`
+  You can use the rows property to set the max height of the Textarea. It will begin to overflow after the rows limit has been reached`}
```

Comment:
This doesn't really capture that `rows` sets both the min and max height. Consider something like
"The `rows` prop sets the number of rows shown in `TextArea`. The input will show a scrollbar if the content exceeds the `rows` limit."